### ISSUE TITLE
Can't open a second channel to a node

### ISSUE THREAD

**Issue by s-tikhomirov** [ElementsProject/lightning#3454]
### Issue and Steps to Reproduce

I'm testing a local LN setup on testnet. I opened a channel but can't use it (see #3452). I decided to not close the existing channels, but rather open a new (public) one. Turns out it is not possible to open a channel to a peer of a channel already exists between them:
```
lightning-cli fundchannel -k id=024c2d485bab58bfde4578937bb16160f87cb91868fe2fefc238ea53455e4d7244 amount=100000 announce=true push_msat=30000
{
   "code": -1,
   "message": "Peer already CHANNELD_NORMAL"
}
```

Why is this the case? I thought BOLTs don't prohibit such behavior, at least based on the fact that BOLT4 describes [non-strict forwarding](https://github.com/lightningnetwork/lightning-rfc/blob/7c1edeb063086b75696e55035b5cba1731642141/04-onion-routing.md#non-strict-forwarding), which assumes that there may be multiple channels between a pair of nodes.

### `getinfo` output
```
{
   "id": "02f67d5b7185ac67172807fc8f9d6b000305e93793332ed5a2087478a69909371e",
   "alias": "clightningB",
   "color": "02f67d",
   "num_peers": 3,
   "num_pending_channels": 0,
   "num_active_channels": 2,
   "num_inactive_channels": 0,
   "address": [],
   "binding": [
      {
         "type": "ipv4",
         "address": "127.0.0.1",
         "port": 9740
      }
   ],
   "version": "v0.8.0-40-g899f5de",
   "blockheight": 1664150,
   "network": "testnet",
   "msatoshi_fees_collected": 0,
   "fees_collected_msat": "0msat",
   "lightning-dir": "/home/sergei/.lightningB/testnet"
}
```

**Comment by cdecker** [comment]
This has come up a couple of times (see https://github.com/ElementsProject/lightning/issues/2001 and https://github.com/ElementsProject/lightning/issues/1389):

> There are a number of decisions that drove us to postpone multiple channels per peer:
> 
>  - It's just simpler to implement a single channel per peer, since there needs to be no multiplexing of channels over a single connection.
> - Users shouldn't really care with whom they have a channel open, as long as the destination is reachable, directly or indirectly. If direct channels are required we might as well give up.
> - Setting up dual-funded channels (which will be on the table for the November spec meeting) is far better than setting up multiple channels to achieve the same goal. Dual funded channels have half of the on-chain footprint, and have the sum total of capacities at their disposal rather than partitioning across multiple channels.
> - The same thing can be achieved via splicing (also going to be discussed in November), i.e., set up a single-funded channel, then have the other side splice some funds in, and you just bumped the capacity of the single channel (no need to increase the network graph size).
>
> That being said, it still is on the roadmap, just not the highest priority issue :wink:

I think the general feeling is that opening multiple channels with the same peer should be discouraged due to it's effect of increasing reliance on a single peer, while there is very little upside, as compared to opening channels with a more diverse set of peers, and then using MPP to perform payments in parallel over multiple peers at the same time.

**Comment by ZmnSCPxj** [comment]
Propose closing as duplicate of #2001

**Comment by s-tikhomirov** [comment]
Thanks for the clarification, closing.
